server.js

```diff
@@ -6129,7 +6129,7 @@ cache(function(data, match, sendBadge, request) {
       badgeData.text[0] = data.label || ghLabel;
       badgeData.text[1] = '' + count;
       badgeData.colorscheme = null;
-      badgeData.colorB = '#' + (color || '78bdf2');
+      badgeData.colorB =  '#' + (data.colorB || color || '78bdf2');
```

Comment:
This won't work if `colorB` is something like `lightgreen`.

There's a function `makeColor` which you could import from `./lib/badge-data` which would help.

Reply:
I have changed it. but  is this `?colorB=black`  supported ? 😮

Reply:
Haha, you're totally right, it's not. I guess the website badge does, but not the query parameters. There's always more to learn about this project!